### DISCUSSION TITLE
aws filtering not working

### DISCUSSION THREAD

**Discussion by 2ffs2nns** [aquasecurity/trivy#6539] [Q&A] [answered]
### Question

has anyone been able to filter aws issues? I can't get `" --ignorefile .trivyignore"` or `"--ignorefile ./.trivyignore.yaml"` to work, and I can't find where this feature was pulled into either v0.50.1 or v0.50.2

off of main, I see
```
feat(aws): apply filter options to result (#6367)
commit 2d8d63e61a43e9ac959d82643e6367daabaa9d73
```
but I don't see it v0.50.2

https://github.com/aquasecurity/trivy/blame/v0.50.2/pkg/cloud/report/report.go
vs
https://github.com/aquasecurity/trivy/blame/main/pkg/cloud/report/report.go

### Target

AWS

### Scanner

Misconfiguration

### Output Format

JSON

### Mode

Standalone

### Operating System

ventura 13.6.6

### Version

```bash
trivy --version                                                                                                                  
Version: 0.50.2
Vulnerability DB:
  Version: 2
  UpdatedAt: 2024-04-22 12:11:55.012425743 +0000 UTC
  NextUpdate: 2024-04-22 18:11:55.012425233 +0000 UTC
  DownloadedAt: 2024-04-22 17:28:55.407448 +0000 UTC
Policy Bundle:
  Digest: sha256:aa1640957b796d93a0ffc5d91237ee6b7ed9467b8f1825279384d29f91b9e590
  DownloadedAt: 2024-04-22 16:37:26.171286 +0000 UTC
```

**Comment by simar7** [comment, marked as answer]
hi @2ffs2nns it will be released in v0.51.0 as patch releases are limited to bug fixes only. Stay tuned!

**Reply by 2ffs2nns** [reply]
thank you!
